blogs/oadp-odf/oadp-blog.md

```diff
@@ -0,0 +1,260 @@
+# Guide: back up and restore persistent workloads on OpenShift using OADP and ODF
+<!--
+We want to publish a blog that contains a guided example of backing up and restoring a CSI-based workload. This should contain:
+
+-   OADP Overview
+-   Installation
+-   Application Overview
+-   Backup/Restore of app showing CSI functionality
+-   Upcoming features (Data Mover)
+-->
+
+OpenShift API for Data Protection (OADP) enables backup, restore, and disaster recovery of applications on an OpenShift cluster. Data that can be protected with OADP include k8s resource objects, persistent volumes, and internal images.
+The OpenShift API for Data Protection (OADP) is designed to protect Application Workloads on a single OpenShift cluster.
+
+Red Hat® OpenShift® Data Foundation—previously Red Hat OpenShift Container Storage—is software-defined storage for containers. Engineered as the data and storage services platform for Red Hat OpenShift, Red Hat OpenShift Data Foundation helps teams develop and deploy applications quickly and efficiently across clouds.
+
+In this guide, we will cover:
+- Operators installation
+  <!-- - Local Storage Operator -->
+  - OpenShift Data Foundation
+  - OpenShift API for Data Protection Operator
+- Application deployment
+- Application protection
+- A disaster scenario
+- Application recovery from disaster
+## Pre-requisites
+- Terminal environment
+  - Your terminal has the following commands
+    - [oc](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/cli_tools/openshift-cli-oc) binary
+    - [git](https://git-scm.com/downloads) binary
+  - Alternatively enter prepared environment in your terminal with `docker run -it ghcr.io/kaovilai/oadp-cli:v1.0.1 bash`
```

Comment:
This is cool. Maybe include `velero` binary too.

TBH you can also likely just assume someone has the tools installed too.

Reply:
I was trying to get away with less. I actually in the cli container include a velero alias which uses velero command from velero server on cluster to
- save on downloads
- eliminate version mismatch between client cli/server

I can add an alias command as a note for velero.